src/main/resources/org/jenkinsci/plugins/authorizeproject/form/dropdownDescriptorSelector.jelly

```diff
@@ -66,7 +66,7 @@ THE SOFTWARE.
       <f:dropdownListBlock value="${loop.index}" title="${descriptor.displayName}"
                            selected="${current.descriptor==descriptor || (current==null and descriptor==attrs.default)}"
                            staplerClass="${descriptor.clazz.name}"
-                           lazy="${attrs.lazy?attrs.lazy+',':''}descriptor,it">
+                           lazy="${(attrs.capture != null and attrs.capture != '')?attrs.capture+',':''}descriptor,it">
```

Comment:
`value?true:false` looks return `false` even value is defined.
That behavir differs from `value?:''` and confusing...